apidoc/Titanium/UI/View.yml

```diff
@@ -404,9 +404,15 @@ events:
       - name: force
         summary: |
             The current force value of the touch event.
-            Note: This property is only available for iOS devices that support 3D-Touch and run 9.0 or later.
+            Note: This property is only available for iOS devices that support 3D-Touch and run 9.0 or later and on some android devices
```

Comment:
"some android devices" sounds a bit odd, can you describe that a bit better? Also please capitalize the "Android" and set the period (`.`) at the end of the sentence.